client-app/ui-kit/components/molecules/address-form/vc-address-form.vue

```diff
@@ -157,6 +170,7 @@ const emit = defineEmits<{
 
 const _emptyAddress: Readonly<MemberAddressType> = {
   isDefault: false,
+  addressType: 3,
```

Comment:
No need. A situation may arise where the address to be edited is of type "2" and the "clear" function will change it to "3".

Reply:
Going to add address type as a property for the form. Checkout cases are about it - shipping and billing addresses.